index.html

```diff
@@ -1028,6 +1061,461 @@ <h2>
+        modify the definition of a [=valid effect=] for effect types defined
+        in this specification.
+      </p>
+      <pre class="idl">
+        enum GamepadHapticEffectType {
+          "dual-rumble"
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn data-for="GamepadHapticEffectType">"dual-rumble" effect
+          type</dfn>
+        </dt>
+        <dd>
+          <p>
+            {{GamepadHapticEffectType/"dual-rumble"}} describes a haptics
+            configuration with an eccentric rotating mass (ERM) vibration
+            motor in each handle of a standard gamepad. In this
+            configuration, either motor is capable of vibrating the whole
+            gamepad. The vibration effects created by each motor are unequal
+            so that the effects of each can be combined to create more
+            complex haptic effects.
+          </p>
+          <p>
+            A {{GamepadHapticEffectType/"dual-rumble"}} effect is a
+            fixed-length, constant-intensity vibration effect intended for an
+            actuator of this type. {{GamepadHapticEffectType/"dual-rumble"}}
+            effects are defined by {{GamepadEffectParameters/startDelay}},
+            {{GamepadEffectParameters/duration}},
+            {{GamepadEffectParameters/strongMagnitude}}, and
+            {{GamepadEffectParameters/weakMagnitude}}, none of which are
+            required.
+          </p>
+          <p>
+            {{GamepadEffectParameters/startDelay}} and
+            {{GamepadEffectParameters/duration}} set the duration of the
+            initial delay before after initiating the effect before the
+            effect will be started
+          </p>
+          <p>
+            {{GamepadEffectParameters/startDelay}} sets the duration of the
+            delay after {{GamepadHapticActuator/playEffect()}} is called
+            until vibration is started, in milliseconds. During the delay
+            interval, the actuator must not vibrate. An effect with negative
+            {{GamepadEffectParameters/startDelay}} is not a [=valid effect=].
+            If the {{GamepadEffectParameters/startDelay}} is not specified,
+            the default delay is 0 milliseconds.
+          </p>
+          <p>
+            {{GamepadEffectParameters/duration}} sets the duration in
+            milliseconds that the actuator will vibrate. If
+            {{GamepadEffectParameters/duration}} is not specified, the
+            default duration is 0 milliseconds.
+          </p>
+          <p>
+            {{GamepadEffectParameters/strongMagnitude}} and
+            {{GamepadEffectParameters/weakMagnitude}} set the vibration
+            intensity levels for the low-frequency and high-frequency rumble
+            motors, normalized to the range `[0,1]`, default 0. An effect
+            with magnitude values outside of this range is not a [=valid
+            effect=].
```

Comment:
As above... this should be generalized... it's not specific to "dual-rumble" 

```suggestion
            {{GamepadEffectParameters/strongMagnitude}} and
            {{GamepadEffectParameters/weakMagnitude}} set the 
            intensity levels for the low-frequency and high-frequency vibrations, 
            normalized to the range `[0,1]`, defaulting to 0. An effect
            with magnitude values outside of this range is not a [=valid
            effect=].
```

The " An effect with magnitude values outside of this range is not a [=valid effect=]" should be checked as part of a valuation algorithm.

Reply:
Thanks! The magnitude values are being checked as part of the validation algorithm in the paragraph below.